libs/fixture-generators/bin/generate-cvr-fixtures

```diff
@@ -4,18 +4,14 @@
 
 export NODE_ENV=development
 
-rm -rf ../fixtures/data/electionGridLayoutNewHampshireTestBallot/castVoteRecords/generated
-./bin/generate-cvrs --electionDefinition ../fixtures/data/electionGridLayoutNewHampshireTestBallot/election.json --outputPath ../fixtures/data/electionGridLayoutNewHampshireTestBallot/castVoteRecords/generated
+rm -rf ../fixtures/data/electionGridLayoutNewHampshireTestBallot/castVoteRecords/machine_0000__2024-01-01_00-00-00
```

Comment:
Had to rename the CVR fixture directory names to ensure that they pass the artifact auth check of machine ID